parent/pom.xml

```diff
@@ -222,6 +223,15 @@
               <goal>manifest</goal>
             </goals>
           </execution>
+          <execution>
+            <id>bundle-baseline</id>
+            <goals>
+              <goal>baseline</goal>
+            </goals>
+            <configuration>
+                <comparisonVersion>1.7.36</comparisonVersion>
```

Comment:
Using 1.7.36 is good for the initial verification but eventually it should be the previous release to ensure there is no unintended future breakage.
With a 1.7 baseline checking is pointless since we had a major version bump compared to it, so everything is allowed.

Reply:
Not really, because only the 1.8 package versions are properly semantically versioned. The 2.x line isn't as there each package just inherits the bundle/project version. What matters is only that 1.8 is correctly semantically versioned against the last 1.x version. Introducing a check against the latest 2.x release would be orthogonal to this use case (and would only make sure that the 2.x package versions are also semantically correct which isn't the case yet)